bitcoin/src/blockdata/constants.rs

```diff
@@ -54,6 +54,8 @@ pub const MAX_SCRIPT_ELEMENT_SIZE: usize = 520;
 pub const SUBSIDY_HALVING_INTERVAL: u32 = 210_000;
 /// Maximum allowed value for an integer in Script.
 pub const MAX_SCRIPTNUM_VALUE: u32 = 0x80000000; // 2^31
+/// Number of blocks needed for an output from a coinbase transaction to be spendable.
+pub const COINBASE_MATURITY: u32 = 100;
```

Comment:
It kinda feels like this should be `locktime::relative::Height`

Reply:
Oops, I hit merge before seeing this.

But this is an interesting observation. I don't think I agree -- locktimes are a semantic part of transactions while this number is a blockheight delta.

Reply:
Good observation.

Reply:
No problem, didn't want to block over this anyway.

My thinking is more like "type that represents a relative quantity of blocks" somewhat similar to `std::time::Duration`, but since `Height` is within `locktime` it also has "intended for timelocks" property. Maybe it'd make sense to have `Height` in `units` for it to be broader, so lock time types would be only the enums. Then if someone is writing an algorithm that does something with relative number of blocks it could be useful.

But it's certainly possible I'm overcomplicating this.

Reply:
We'd loose `relative::Height` and `absolute::Height` if we did that, if that matters.

Reply:
How so? Nothing prevents us from having `units::relative::Height`.

Reply:
Fair.